src/data/analyse_en.json

```diff
@@ -124,10 +124,10 @@
+                "content": "<strong>Warnings received by data donors</strong><p>Count of red and green risk warnings as well as the sum of both risk warnings received by data donors in the Corona-Warn-App.</p><strong>Time intervals</strong><p>Daily: Count accurate to the day<br />Weekly: Weekly count by calendar week (CW)<br />7-day moving average: 7-day moving average accurate to the day<br />Cumulative: Cumulative sum accurate to the day, since the beginning of the measurement</p><strong> Reference date</strong><p>The reference date is the “submitted_at” date of the PPA dataset.</p><strong>Data sources</strong><p>Privacy preserving analytics (PPA)</p><strong>Additional information</strong><p>Users of the Corona-Warn-App have the opportunity to participate in the data donation and make an important contribution to improve the app. By voluntarily sharing selected data, users can help the Robert Koch Institute to evaluate the effectiveness of the Corona-Warn-App and improve the features and usability of the app. To be a data donor, the switch in the settings under "Share data" must be set to "On".</p><p>Users who have had a brief and/or distant encounter with a person who was tested positive for corona in the past 14 days, receive a green risk warning. This means that the encounter was assumed not to be above the defined threshold value. There is no need for specific action from the users concerned.</p><p>Users who have had long and/or close encounters with at least one person who tested positive for corona in the past 14 days, receive a red risk warning. This means that the encounter was assumed to be above the defined threshold. Affected users are asked to reduce personal contacts. In case of symptoms, they should consult a general practitioner, the medical on-call service, or the local health department. The general practitioner or the local health department will decide whether a test will be provided.</p>"
```

Comment:
Escape the quotes like this:

\"Share data\" must be set to \"On\"

Reply:
@MikeMcC399 
Thank you very much for your suggestion. 
I've escaped the quotes as you described it.
As editor I am using Notepad++ 

Best regards
Tobias